src/index.js

```diff
@@ -85,6 +86,18 @@ module.exports = function SitemapGenerator(uri, opts) {
     ) {
       emitter.emit('ignore', url);
     } else {
+      // https://zendesk.atlassian.net/browse/WT-5268 - ignore canonicalized pages
+      if (options.ignoreCanonacalized) {
+        const canonicalMatches = /<link rel="canonical" href="([^"]*)"/gi.exec(page);
+        if (canonicalMatches && canonicalMatches.length > 1) {
+          const canonical = matches[1];
+          if (canonical && canonical !== url) {
```

Comment:
I considered making this case-insensitive, but [Scott's spreadsheet](https://docs.google.com/spreadsheets/d/1TaL5cw5s-pms8sUDpwWGpb63EeGVskObS07pzEF6tK4/edit#gid=1551178505) lists some pages that only differ by capitalization, so I'm erring on the side of matching the spreadsheet.